src/mlpack/core/data/load_arff_impl.hpp

```diff
@@ -103,7 +102,7 @@ void LoadARFF(const std::string& filename,
           // `origDimType` string here instead (which has not had ::tolower used
           // on it).
           types.push_back(true);
-          boost::trim_if(origDimType,
+	  trim_if(origDimType,
```

Comment:
I think the spacing is wrong here.  Maybe your editor is using tabs instead of spaces?  I can't fully see it in the github diff here, but it's probably worth looking into if this looks right in your editor but incorrect here.

Reply:
It was looking right in my editor(Vim).
I added the tab settings as mentioned on the mlpack style guidelines page.
Also reindented the part. I hope it works.